tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java

```diff
@@ -79,19 +109,21 @@ public long timeTillKick() {
         final boolean guest = (user == null);
         long myTimeout; // timeout in seconds.
 
-        if (user != null && UserRegistry.userIsTC(user)) {
-            myTimeout = 60 * 20; // 20 minutes
-        } else if (guest) {
+        if (guest) {
+            myTimeout = Params.CLDR_GUEST_TIMEOUT.value();
```

Comment:
The code is difficult to understand, because sometimes variables appear to be in seconds, and sometimes fields appear to be in millis.

It would be clearer if the function names and variables were always crystal clear as to what the units were. Eg instead of the name timeTillKick, it should be millisTillKick if it is supposed to return millis, and secondsTillKick if it is supposed to return seconds. Same with CLDR_GUEST_TIMEOUT. Either CLDR_GUEST_TIMEOUT_MILLIS or CLDR_GUEST_TIMEOUT_SECS, depending on what it is supposed to contain. Same with myTimeoutMillis, etc. that way if we see one item returned or assigned to another, it is clear that if they are not the same scale, there must be a factor to convert them.

Reply:
OK, I'll do a bunch of renaming.